### ISSUE TITLE
Repo organization / cleanup

### ISSUE THREAD

**Issue by trvrb** [blab/mpox-dynamics#3]
A few points that could use cleanup / could be better organized:

1. I went to `/data` expecting to find sequence FASTAs, but instead only find ancillary data. Looking more it's clear that core sequence data is in `dta/alignments/` and `mascot_glm/alignments/`. Rename `/data` to something more descriptive of move it to live in the appropriate analysis subdirectory.
2. Get ride of all the `.DS_Store` cruft. This should be listed in your `.gitignore` file. I see it is. Perhaps these snuck in before you added to `.gitignore`. You should just manually delete all these `.DS_Store` files from the repo now.
3. Similarly, manually delete all the `.Rhistory` files. Add `.Rhistory` to `.gitignore`.
4. Manually delete all the `.ipynb_checkpoints/` directories.
5. `sequence_information.zip` is a complete duplicate of `sequence_information/`. Just delete the `.zip` file. Also rename `sequence_information/` to `sequence_acknowledgments`. I expected "sequence information" to mean metadata.
6. `out/` could be named something more informative. I can't tell what in here.
7. Add an repo organization overview to your top-level README file. https://github.com/blab/dengue-antigenic-dynamics and https://github.com/blab/global-migration are pretty good examples of this.
8. Add more intermediate READMEs to coach people through. The key places here are `/dta` and `/mascot-glm`. Again see https://github.com/blab/dengue-antigenic-dynamics/tree/master/titer_model or https://github.com/blab/global-migration/tree/master/geo for decent examples.
9. Ideally (but potentially more a stretch), provide a single `.ipynb` for each figure ala https://github.com/blab/adaptive-evolution/blob/master/adaptive-evolution-analysis/Figure1.ipynb

**Comment by miparedes** [comment]
@trvrb  Thanks for these suggestions! 

I've gone ahead and addressed 1-8 and have updated the main branch accordingly. I'll start working on #9 but it might take a bit of time. Given the time crunch to return to the editor sooner rather than later, I think the best course might be to do it while the editor revises the updated submission. Unless you think it's something that needs to be done before resubmitting?

**Comment by jobsxing** [comment]
@trvrb 

Hi Trevor, thank you for your insightful and valuable contribution in this work. I have one question regarding the repository.

In the README file, it's mentioned that “the maximum clade credibility trees and effective population sizes can be analyzed [here](https://github.com/blab/mpox-dynamics/blob/main/scripts/glm_plotting_and_ne.ipynb), and the percentage of cases due to introductions and Rt over time can be analyzed [here](https://github.com/blab/mpox-dynamics/blob/main/scripts/percent_cases_from_intros.ipynb)”.

However, I couldn’t find the percent_cases_from_intros.ipynb notebook in the current version of the repository.

Since this part of the result seems quite important for interpreting the findings of the manuscript, I was wondering if you could kindly check whether this notebook is missing or has been moved elsewhere?

Thank you again for your work and time! Looking forward to your reply!
